pyspecProcScripts/DCCT_func.py

```diff
@@ -223,7 +225,7 @@ def place_labels(ax1, label, label_placed, this_label_num, check_for_label_num =
                 # then searching for BBox docs
                 logging.debug(strm( "tick locations",[j.get_window_extent().bounds for j in ax1.get_yticklabels()] ))
                 x_textdisp = [j.get_window_extent().bounds for j in ax1.get_yticklabels()][0][0]
-            x_textdisp -= 3.7*text_height - arrow_width_px
```

Comment:
this was hardset to 3.7 when I change it though the labels seem to move correctly